app/models/category.rb

```diff
@@ -4,20 +4,16 @@ class Category < ApplicationRecord
   has_and_belongs_to_many :tags, class_name: 'ActsAsTaggableOn::Tag'
 
   validates :name, presence: true
-  translates :name, fallbacks_for_empty_translations: true
+  translates :name, touch: true, fallbacks_for_empty_translations: true
```

Comment:
When you add touch: true to a translates association, it ensures that updating a translation will also update the parent model’s updated_at timestamp.

Why Use touch: true?

✔ Keeps cache invalidation correct (if you cache based on updated_at).
✔ Ensures database triggers or background jobs detect updates.
✔ Prevents stale data issues when translations change but updated_at doesn’t update.